docs/concepts/capabilities.mdx

```diff
@@ -56,16 +60,16 @@ We believe a healthy wasmCloud ecosystem includes multiple implementations of ca
 
 [host]: https://github.com/wasmCloud/wasmcloud
 
-# Capability Providers
+### Capability Providers
 
 In wasmCloud, _capability providers_ are self-contained processes that implement a _capability contract_, which corresponds to a well-defined API.
 
 Capability providers run on your wasmCloud lattice, and ensure that functionality depended on by actors is available.
 
-## Securing Capability access
-
-Not every actor should be able to access every capability provider. Securing and restricting communication between actors and providers is important to avoid actors gaining access to providers they should not have access to.
+### Deny-by-default Capability Access
 
-To contact a provider that implements a capability contract, an actor must be signed with a the appropriate capability contract ID.
+Not every actor should be able to access every capability. Securing and restricting communication between actors and providers is important to avoid actors gaining access to capabilities they should not have access to.
 
-For example, an actor signed with the contract ID `wasmcloud:keyvalue` will be able to access any `wasmcloud:keyvalue` capability providers running on-lattice.
+To contact a particular capability provider, an actor must:
+- be [signed](/docs/hosts/security) with a the appropriate capability contract ID, which happens automatically during the actor build process
+- be [linked at runtime](./runtime-linking.mdx) to the capability provider
```

Comment:
Should this link be absolute from `/docs` ? Not sure if we have a preference on this, but making it consistent might help with search/replace

Reply:
Docusaurus should catch it if it fails, but agreed on consistency. Absolute URLs with no file extension is preferred. The markdown language server understands file paths and should help with correcting links if anything is moved.

```suggestion
- be [linked at runtime](/docs/concepts/runtime-linking) to the capability provider
```